libctru/source/system/allocateHeaps.c

```diff
@@ -17,8 +19,13 @@ void __attribute__((weak)) __system_allocateHeaps() {
 
 	// Allocate the linear heap
 	svcControlMemory(&__linear_heap, 0x0, 0x0, __linear_heap_size, MEMOP_ALLOC_LINEAR, MEMPERM_READ | MEMPERM_WRITE);
+
+	// Set up the mappable heap
+	__mappable_heap = 0x10000000;
```

Comment:
I think these variables are unnecessary. It's best to hardcode this in mappableInit.